client/src/components/timeline/revision.js

```diff
@@ -7,6 +7,32 @@ import Wiki from 'app/entities/wiki';
 import Date from './date';
 import Timelapse from './timelapse';
 
+// eslint-disable-next-line no-useless-escape
+const REGEX_EDIT_SUMMARY_PARTS = /(?:\/\*([^\*]+)\*\/)?(.+)?/;
+
+// Append section name to the page title if it is included in the edit summary
+const getDisplayTitle = ( title, comment ) => {
+	const matches = REGEX_EDIT_SUMMARY_PARTS.exec( comment );
```

Comment:
You could also use [comment.match(REGEX_EDIT_SUMMARY_PARTS)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match), which is probably more common, but this is fine.

Reply:
I'll change it if you like it more